### PR TITLE
Update usage-analytics.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another FAQ for about usage summary reports reader role.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4798

File: microsoft-365/admin/usage-analytics/usage-analytics.md [hunk 1]
```diff
@@ -84,6 +84,9 @@ User activity reports are available for certain individual services. These repor
 It is not free, you will need a Power BI Pro license. For details see [prerequisites](/power-bi/service-template-apps-install-distribute#prerequisites) for installing, customizing, and distributing a template app.
 
 To share the dashboards with others, please see more at [Share dashboards and reports](/power-bi/service-how-to-collaborate-distribute-dashboards-reports#share-dashboards-and-reports).
+### Is Usage summary reports reader role enough to view the unage analytics?
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/usage-analytics.md]:
```suggestion
### Is the Usage Summary Reports Reader role enough to view the usage analytics?
```

File: microsoft-365/admin/usage-analytics/usage-analytics.md [hunk 2]
```diff
@@ -84,6 +84,9 @@ User activity reports are available for certain individual services. These repor
...
+
+Usage Summary Reports Reader role gives them access to only tenant level aggregates in M365 Usage Analytics.  Microsoft recommends Reports Reader or Usage Summary Reports Reader role to anyone who's responsible for change management and adoption, but not necessarily an IT administrator.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/usage-analytics/usage-analytics.md]:
```suggestion
The Usage Summary Reports Reader role only allows access to tenant level aggregates in Microsoft 365 usage analytics.  We recommend the Reports Reader or Usage Summary Reports Reader role to anyone who's responsible for change management and adoption, but is not necessarily an IT administrator.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this PR. Thanks

**Comment by efrene** [comment]
This has been merged. Thanks.

**Pull request merged**